activemodel/lib/active_model/type/helpers/time_value.rb

```diff
@@ -50,18 +50,18 @@ def user_input_in_time_zone(value)
 
         private
 
-          def new_time(year, mon, mday, hour, min, sec, microsec, offset = nil)
+          def new_time(year, mon, mday, hour, min, sec, microsecond, offset = nil)
```

Comment:
This change isn’t worth it. The other variables would have to be renamed as well. But let’s keep the git history here.

Reply:
@kaspth reverted this change.